zc_common/jwt_auth/authentication.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, **kwargs):
             setattr(self, kwarg, kwargs[kwarg])
 
     def is_authenticated(self):
-        return 'user' in self.roles or 'service' in self.roles
+        return True
```

Comment:
It was in response to this sentence
> Also, I am returning True because all requests must be authenticated by 'JWTAuthentication' class.

As for whether a user is anonymous or not, that has no bearing on this. The reason the `User` class has the `is_authenticated()` function in the first place is so that we maintain similarity with Django's `User` model as much as we can. Forcing true defeats the entire purpose. If it's always True, then you might as well just delete the function since it makes no difference.